### PR TITLE
View all grades - Feature 1 | Ryan Griego

### PR DESCRIPTION
**Pull request by ryan-griego**

### CODE REVIEW

**Comment by ryan-griego** [comment]
Screen Shot 2020-05-06 at 7 29 45 PM

File: components/app.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  }
+
+  handleGetGradesError(error) {
+    console.error(error);
+    console.log("Error");
+  }
+
+  handleGetGradesSuccess(grades) {
+    console.log("log grades", grades);
+    console.log("Success");
```
Comment (scbowler) [inline components/app.js:16]: Make sure to remove `console.log`s once you are done with them and before making a pull request to master. Production code shouldn't have `console.log`s in it, the only exception being `console.error` for errors.

File: components/app.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+class App {
+  constructor(gradeTable) {
+    this.gradeTable = gradeTable;
+    this.handleGetGradesError = this.handleGetGradesError.bind(this);
+    this.handleGetGradesSuccess = this.handleGetGradesSuccess.bind(this);
+  }
+
+  handleGetGradesError(error) {
+    console.error(error);
+    console.log("Error");
```
Comment (scbowler) [inline components/app.js:11]: Remove, see comment on line 16.

File: main.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+table = document.querySelector('table');
+console.log(table);
```
Comment (scbowler) [inline main.js:2]: Remove

File: main.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+var newGradeTable = new GradeTable(table);
+console.log("log newGradeTable", newGradeTable);
```
Comment (scbowler) [inline main.js:5]: Remove

File: main.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+var newApp = new App(newGradeTable);
+console.log("log newApp", newApp);
```
Comment (scbowler) [inline main.js:8]: Remove

**Review by scbowler — APPROVED** [review]
Looks really good! 👍 

Glad to see you are using `console.log`s a lot to verify things are what you expect them to be that is really good! However once your code is working and before making a Pull Request you want to remove them, they have served their purpose, remember you can always add one back in if you need to check anything again down the road. You don't need to change anything for this Pull Request just keep that in mind moving forward. Once again great job on everything else!

**Pull request merged**